src/utils/postAttestationProfile.js

```
@@ -0,0 +1,88 @@
+
+    if (!Validation.isDataObject(profile)) {
+        throw new ErrorWithMessage('Invalid data object', { code: "INVALID_DATA" });
+    }
+
+    return new Promise((resolve, reject) => {
+        if (!attestorAddress) {
+            return reject('Attestor address not available');
+        }
+
+        function onError(err) {
+            logger.error('(postAttestationProfile): ' + err);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)